### PR TITLE
Update for DP resilience new changes

### PR DESCRIPTION
**Pull request by StarlightIbuki**
### Description

https://github.com/Kong/kong-ee/pull/7627

KAG-2949

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Review by lena-larionova — CHANGES_REQUESTED** [review]

File: app/_src/gateway/kong-enterprise/cp-outage-handling.md [hunk 1]
```diff
@@ -20,15 +20,19 @@ This option is only recommended for customers who have to adhere to strict avail
...
+The backup node is responsible for communicating the state of the {{site.base_gateway}} `kong.conf` configuration file from the control plane to the S3 bucket.
 
 Nodes are initialized with fallback configs via environment variables, including `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_DEFAULT_REGION`. 
 If associating with an IAM role and if the backup node does not reside on the AWS platform, an additional environment variable `AWS_SESSION_TOKEN` may be necessary. 
 
-A backup node should not be used to proxy traffic. A single backup node is sufficient for all deployments. For more information about the data that is set in the environment variables, review the [AWS environment variable configuration documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).
+We do not recommend using backup nodes to proxy traffic though it's possible. The backup job enlarges the attack surface of a proxying DP, and contribute significantly to the P99 delay. You need to know the risk if you want to deloy it this way, and the DP needs to be at least `3.6.0.0` to be provisioned with backup configuration when it's configured as a backup node.
```
[the remark this answers is not in the document]
Comment (StarlightIbuki) [reply app/_src/gateway/kong-enterprise/cp-outage-handling.md:29]: The risk also applies to versions before 3.5. Not sure if we should change the wording for older versions.

File: app/_src/gateway/kong-enterprise/cp-outage-handling.md [hunk 2]
```diff
@@ -46,7 +50,11 @@ kong-exporter:
 
 ```
 
-This node is responsible for writing to the S3 bucket when it receives new configuration. The file structure is automatically created inside of the bucket and should not be created manually. If the node version is `3.2.0.0`, using the example above, the key name will be `test-prefix/3.2.0.0/config.json`. 
+All the object keynames/prefixes mentioned in the following paragraphs are parametized with the prefix given in the config and the gateway version. Let's say the node has a version of `3.6.0.0`,
```
[the remark this answers is not in the document]
Comment (StarlightIbuki) [reply app/_src/gateway/kong-enterprise/cp-outage-handling.md:53]: I've rephrased the paragraph and some of the sentences are moved below.

File: app/_src/gateway/kong-enterprise/cp-outage-handling.md [hunk 3]
```diff
@@ -20,15 +20,30 @@ This option is only recommended for customers who have to adhere to strict avail
...
-This node is responsible for communicating the state of the {{site.base_gateway}} `kong.conf` configuration file from the control plane to the S3 bucket.
+The backup node must have read and write access to the S3 bucket, and the data plane nodes that are provisioned must have read access to the same S3 bucket.
+The backup node is responsible for communicating the state of the {{site.base_gateway}} `kong.conf` configuration file from the control plane to the S3 bucket.
 
 Nodes are initialized with fallback configs via environment variables, including `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_DEFAULT_REGION`. 
 If associating with an IAM role and if the backup node does not reside on the AWS platform, an additional environment variable `AWS_SESSION_TOKEN` may be necessary. 
 
+{% if_version lte:3.5.x %}
+
 A backup node should not be used to proxy traffic. A single backup node is sufficient for all deployments. For more information about the data that is set in the environment variables, review the [AWS environment variable configuration documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).
```
Comment (StarlightIbuki) [inline app/_src/gateway/kong-enterprise/cp-outage-handling.md:31]:
```suggestion
{:.important}
> We do not recommend using backup nodes to proxy traffic. The backup job enlarges the attack surface of a proxying DP and contributes significantly to the P99 delay. You need to know the risk if you want to deploy a node this way, and a DP acting as a backup node cannot be provisioned with backup configurations.
```

File: app/_src/gateway/kong-enterprise/cp-outage-handling.md [hunk 4]
```diff
@@ -46,7 +59,13 @@ kong-exporter:
 
 ```
 
-This node is responsible for writing to the S3 bucket when it receives new configuration. The file structure is automatically created inside of the bucket and should not be created manually. If the node version is `3.2.0.0`, using the example above, the key name will be `test-prefix/3.2.0.0/config.json`. 
+All the object keynames/prefixes mentioned in the following paragraphs are parametized with the prefix given in the config and the gateway version. Let's say the node has a version of `3.6.0.0`,
```
Comment (lena-larionova) [inline app/_src/gateway/kong-enterprise/cp-outage-handling.md:62]:
```suggestion
{% if_version lte:3.5.x %}
This node is responsible for writing to the S3 bucket when it receives new configuration. The file structure is automatically created inside of the bucket and should not be created manually. If the node version is `3.2.0.0`, using the example above, the key name will be `test-prefix/3.2.0.0/config.json`. 
{% endif_version %}

{% if_version gte:3.6.x %}
All the object keynames/prefixes mentioned in the following paragraphs are parameterized with the prefix given in the config and the gateway version. For example, let's say the node has a version of `3.6.0.0`.
```

**Review by lena-larionova — APPROVED** [review]
LGTM

**Pull request merged**